mangaki/mangaki/admin.py

```diff
@@ -355,6 +358,65 @@ def refresh_work(self, request, queryset):
 
     refresh_work.short_description = "Mettre à jour la fiche de l'anime (poster)"
 
+    @transaction.atomic
+    def change_title(self, request, queryset):
+        if request.POST.get('confirm'):  # Changing default title has been confirmed
+            new_titles = {}
+
+            for key in request.POST:
+                if key.isnumeric():
+                    work_id = int(key)
+                    if request.POST.get(key).isnumeric():
+                        title_id = int(request.POST.get(key))
+                        new_titles[work_id] = title_id
+
+            titles = WorkTitle.objects.filter(
+                pk__in=new_titles.values(), work__id__in=new_titles.keys()
+            ).values_list('title', 'work__title', 'work__id')
+
+            for new_title, current_title, work_id in titles:
+                if new_title != current_title:
+                    Work.objects.filter(pk=work_id).update(title=new_title)
+
+            self.message_user(request, 'Les titres ont bien été changés pour les œuvres sélectionnées.')
```

Comment:
I'm not sure, but I think it's « changé » rather than « changés » due to the presence of « avoir » as a verb.